ui/src/Components/ProfilePage.jsx

```diff
@@ -33,18 +37,66 @@ export default class ProfilePage extends React.Component {
       loading: true,
       profile: null,
       props: props,
+      admin:false,
+      verified:0,
     }
+
+    this.handleReadyParam = this.handleReadyParam.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
   }
 
   async componentDidMount()
   {
-    const url = `https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/viewProfile?id=${this.state.props.match.params.id}`;
+    const url = URL + `id=${this.state.props.match.params.id}`;
     const resp = await fetch(url);
     const data = await resp.json();
-    this.setState({profile: data, loading: false});
+    var adminState = querySearch(this.props.location.search).admin;
+    this.setState({profile: data, loading: false, admin: adminState});
+  }
+
+  handleReadyParam(event){
```

Comment:
It's not immediately obvious what this variable name refers to, to me. Maybe something like "handlePendingVerParam"?